source/common/upstream/health_discovery_service.h

```diff
@@ -41,25 +93,42 @@ class HdsDelegate
       std::unique_ptr<envoy::service::discovery::v2::HealthCheckSpecifier>&& message) override;
   void onReceiveTrailingMetadata(Http::HeaderMapPtr&& metadata) override;
   void onRemoteClose(Grpc::Status::GrpcStatus status, const std::string& message) override;
-
+  void sendResponse();
   // TODO(htuch): Make this configurable or some static.
   const uint32_t RETRY_DELAY_MS = 5000;
+  uint32_t server_response_ms_ = 1000;
+  uint32_t cluster_connection_buffer_limit_bytes = 12345;
+  uint32_t cluster_timeout_s = 1;
```

Comment:
These constants should be consistently capitalized (including RETRU_DELAY_MS). Suggested template is `static constexpr uint32_t ClusterTimeoutSeconds = 1;`